sphinx/tutorial/cylc/runtime/introduction.rst

```diff
@@ -0,0 +1,300 @@
+
+When the :term:`job script` is executed the stdout and stderr are redirected
+into the ``job.out`` and ``job.err`` files which are also stored in the
+:term:`job log directory`.
+
+The :term:`job log directory` lives in a directory structure under the
+:term:`cycle point`, :term:`task` name and the :term:`job submission number`::
+
+   ~/cylc-run/<suite-name>/log/job/<cycle-point>/<task-name>/<job-submission-no>/
+
+The :term:`job submission number` starts at 1 and increments each time a taks
+is re-run.
+
+.. practical::
+
+   .. rubric:: In this practical we will add some scripts to and run the
+      :ref:`weather forecasting suite <tutorial-datetime-cycling-practical>`
+      from the :ref:`scheduling tutorial <tutorial-scheduling>`.
+
+   #. **Create A New Suite.**
+
+      The following command will copy some files for us to work with into a
+      directory called ``dummy-suite``:
+
+      .. code-block:: bash
+
+         rose tutorial cylc-dummy dummy-suite
+
+      In this directory we have the ``suite.rc`` file from the
+      :ref:`weather forecasting suite <tutorial-datetime-cycling-practical>`
+      with some runtime configuration added to it.
+
+      These is also a script called ``get-observations`` located in a bin
+      directory.
+
+   #. **Add The Initial And Final Cycle Points.**
+
+      TODO
+
+   #. **Register The Suite.**
+
+      Register and validate your suite by running the following commands:
+
+      .. code-block:: bash
+
+         cd dummy-forecast
+         cylc reg dummy-forecast $PWD
+         cylc validate dummy-forecast
+
+   #. **Run The Suite.**
+
+      Open the cylc GUI by running the following command:
+
+      .. code-block:: bash
+
+         gcylc dummy-forecast &
+
+      Run the suite by executing the following command:
+
+      .. code-block:: bash
+
+         cylc run dummy-suite
+
+      The tasks will start to run, you should see them going through the
+      waiting, running and succeeded states.
+
+      When the suite reaches the final cycle point and all tasks have succeeded
+      it will shutdown automatically and the GUI will go blank.
+
+      .. tip::
+
+         You can also run a suite from the cylc gui by pressing the "play"
+         button.
+
+         .. image:: ../img/gcylc-play.png
+            :align: center
+
+   #. **Inspect A Job Log.**
+
+      Cylc will have created a :term:`run directory` for this suite in the
+      following location::
+
+         ~/cylc-run/dummy-forecast/
+
+      Try opening the ``job.out`` file for one of the ``get_observations``
+      jobs. The file will be located within the :term:`run directory`::
+
+         <run-directory>/log/job/<cycle-point>/get_observations_heathrow/01/job.out
+
+   #. **Inspect A Work Directory.**
+
+      The ``get_rainfall`` task should create a file called ``rainfall`` in its
+      :term:`work directory`. Try opening the file::
+
+         <run-directory>/work/<cycle-point>/get_rainfall/rainfall
```

Comment:
Should we encourage the users to inspect the suite log as well?

Reply:
Done in the next commit.